app/static/scss/client/blog/post.scss

```diff
@@ -29,4 +29,8 @@
         margin-bottom: 1rem;
         line-height: 1.9rem;
     }
+    ul {
+        list-style-type: circle;
```

Comment:
Can we make this `disc` so that the circles are filled in? Then make a rule for `<ul>`s inside `<ul>`s with `list-style-type: circle` so that second level indentations are not filled in